code/game/objects/items/weapons/syndie.dm

```diff
@@ -53,8 +53,8 @@
 /obj/item/weapon/syndie/c4detonator
```

Comment:
So, what is the real difference between this and the regular c4? Why is the detonator needed?

Reply:
C4 as is can only be set on a timer which is really unrealistic for its use. C4 is used for key detonation at key points on call, not a timebomb